CONTRIBUTING.md

```diff
@@ -233,3 +235,55 @@ _NOTE: with all of these steps, if you have SSH Keys set up, you will want to us
 5. Make a new branch: `git checkout -b your_branch_name`.
 6. Pull in content from you private repo: `git pull private main`.
 7. Push your new work up to GitHub: `git push origin your_branch_name`.
+
+## Troubleshooting
+
+This section attempts to capture some of the more common troubleshooting techniques to try
+if you face issues in contributing.
+
+### General Install, compile or build issues
+
+Many of the common errors you'll face can be resolve by the following:
+
+1. Ensure you are using `Yarn` not `NPM`
+2. When in doubt completely delete the `node_module` folder in the root project directory, and re-run `yarn install`
```

Comment:
```suggestion
2. When in doubt completely delete the `node_modules` folder in the root project directory, and re-run `yarn`
```

You may also consider calling this out as a "last resort". This suggestion is the most time consuming, and usually we shouldn't have to do this. It seems every now and then when we upgrade Gatsby, we need to clear out node modules and reinstall all of them to fix some things, but again, this should be a last resort option and not something we need to do often.

By making this 2nd in the list, I worry this might communicate we'd expect you to do this often.

Reply:
good point